backend/server.js

```diff
@@ -118,6 +118,62 @@ app.get("/top-tracks", async (req, res) => {
 });
 
 
+app.get("/compatible-tracks", async (req, res) => {
+  try {
+    // get the most listened track
+    const topTracksResponse = await spotifyApi.getMyTopTracks({ limit: 1, time_range: 'short_term' });
```

Comment:
if user did not use spotify in the last 4 weeks, this gives error, needs handling